js/router.js

```diff
@@ -298,7 +298,6 @@ module.exports = Backbone.Router.extend({
       throw new Error('At a minimum, the config must contain a config.promise.');
     }
 
-    $('#loadingModal').addClass('hide');
     config = __.extend({}, defaults, config);
 
     this.pageConnectModal && this.pageConnectModal.remove();
```

Comment:
Right below this, the new pageConnectModal should get 

dismissOnEscPress: false

as an option, otherwise you can close it with the esc key, which leaves a blank content area.